app/views/admin/works/_ocr.html.erb

```diff
@@ -0,0 +1,73 @@
+<h3>Summary</h3>
+<div class="m-3">
+    OCR has <% unless @work.ocr_requested %> not <%end%> been requested for this work.
+</div>  
+<div class="m-3">
+  <%= @work.members.count {|m| m.asset? && m.hocr.present?} %>
+  out of 
+  <%= @work.members.count  {|m| m.asset? } %> assets
+  currently have OCR.
+</div>
+
+<h3>OCR table</h3>
+<table class="table admin-list member-list">
+  <thead>
+    <tr>
+      <th class="thumbnail">Thumbnail</th>
+      <th class="type">Type</th>
+      <th class="id">Title</th>
+      <th class="title">OCR</th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% work.members.order(:position).includes(:leaf_representative).each do |member| %>
```

Comment:
I think we talked about paginating here?  I am worried about just putting all 700 (and up!) pages on here, for large pages.  Note these will load with _every_ load of a work/show page -- it's got all tab content on it on every load, whether you choose the tab or not. 

Once we start applying pagination logic here too, I think it starts to give us a hint even more that -- this should be a ViewComponent (that can have it's own methods with logic), not just a partial.  Note that some of hte other tabs on that page are already view components.  In general, we should start always at least considering view components over partials, for all but the simplest stuff. 

Once it's a view component some other stuff can possibly be put into ruby methods in the view component isntead of the template too.  Like the calculation above of `@work.members.count {|m| m.asset? && m.hocr.present?}`, can be in a ruby method in the ViewComponent, `assets_with_hocr_present_count` or something like that.

Reply:
Good call - will refactor!